pyensembl/sequence_data.py

```diff
@@ -20,33 +20,42 @@
 from Bio import SeqIO
 import datacache
 
-from .common import CACHE_SUBDIR, require_ensembl_id
-from .release_info import check_release_number
+from .common import (CACHE_SUBDIR, require_ensembl_id, is_url_format,
+                     copy_to_cache_if_needed, local_file_cache_path)
 
 
 class SequenceData(object):
-    """Container for reference nucleotide and amino acid sequenes.
+    """
+    Container for reference nucleotide and amino acid sequenes.
 
     Downloads and caches FASTA file, creates database mapping
     identifiers to sequences.
     """
     def __init__(
             self,
-            ensembl_release,
-            fasta_url,
+            genome_source,
```

Comment:
Instead of passing a `genome_source` and `fasta_type`, why not just give the fasta path directly?